libs/comments/website/src/lib/comment-list/comment-list-item.tsx

```diff
@@ -33,21 +37,10 @@ export const CommentListItemName = styled('div')`
   font-weight: ${({theme}) => theme.typography.fontWeightBold};
 `
 
-export const CommentListItemVerifiedBadge = styled('div')<{isVerified: boolean; isTeam: boolean}>`
+export const CommentListItemVerifiedBadge = styled('div')`
   display: grid;
   align-items: center;
-
-  ${({theme, isVerified, isTeam}) => css`
```

Comment:
Removed staff badge as that type is when a staff creates a comment in the editor and not for a comment made by that staff.